seed: 123456789
horizon: 6
width: 3
improvement_steps: 6
num_particles: 1000
num_rollouts: 100
num_particles_rollout: 100
gaussian: false
mx: 0.1
my: 0.1
sx: 0.1
sy: 0.1
